src/strategies/MicroGrantsStrategy/types.ts

```diff
@@ -20,7 +20,7 @@ export type Recipient = {
 export type RegisterData = {
   registryAnchor?: `0x${string}`;
   recipientAddress: `0x${string}`;
-  requestedAmount: bigint;
```

Comment:
Let's avoid reusing types between strategies. If they are the exact same, we should move it to a common file.
This will break things on micro grants